### ISSUE TITLE
How to tell generator to generate script name for certain enum members?

### ISSUE THREAD

**Issue by n9** [Saltarelle/SaltarelleWeb#16]
For instance, I would like to specify `[ScriptName("keypress")]` and change the name of `ElementEvents.Keypress` to `ElementEvents.KeyPress`.

**Comment by erik-kallen** [comment]
In what way does `[ScriptName("keypress")] KeyPress` not work?

**Comment by erik-kallen** [comment]
Sorry, thought this was in the compiler repo :)

This does not have any support right now, but you're welcome to add it. I'm not 100% sure, though, that this is a good idea; I see inconsistencies coming where some names are camel-cased and some only have capital first letters. At least the way it is now is consistent.

**Comment by n9** [comment]
I thought that the names could be somehow overridden in `types.meta` to make the naming consistent.

**Comment by erik-kallen** [comment]
Yes, but we'd need to go through every single enum (both current and future ones) and specify the correct casing. IMO not worth the job, the whole idea about auto-generating is to minimize manual work, and I don't see this feature worth the maintenance it requires.
